CruCentralCoast/Community/Ministry Teams/MinistryTeamsTVC.swift

```diff
@@ -50,21 +55,23 @@ class MinistryTeamsTVC: UITableViewController {
             assertionFailure("Probably used the wrong storyboard name or identifier here")
             return
         }
-        vc.configure(with: self.testDataArray[indexPath.row])
         self.navigationController?.present(vc, animated: true, completion: nil)
     }
     
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "comCell", for: indexPath) as! CommunityTableViewCell
         
         
-        // Configure the cell
-        cell.bigLabel.text = testDataArray[indexPath.row].teamTitle
-        cell.bannerImage.image = testDataArray[indexPath.row].teamImage
-        //only returning first team name
-        cell.smallLabel1.text = testDataArray[indexPath.row].teamLeaders[0]
+        let cell = tableView.dequeueCell(CommunityTableViewCell.self, indexPath: indexPath)
+        
+        cell.bigLabel.text = dataArray[indexPath.row].name
```

Comment:
Might be nice to leave all this configuration to the cell class itself. You could assign the MinistryTeam to a property on the cell or call a configuration method on the cell